src/core_atmosphere/Registry.xml

```diff
@@ -395,131 +395,162 @@
-			<var name="meshDensity"/>
-			<var name="bdyMaskCell"/>
-			<var name="bdyMaskEdge"/>
-			<var name="bdyMaskVertex"/>
-			<var name="cf1"/>
-			<var name="cf2"/>
-			<var name="cf3"/>
-			<var name="zgrid"/>
-			<var name="rdzw"/>
-			<var name="dzu"/>
-			<var name="rdzu"/>
-			<var name="fzm"/>
-			<var name="fzp"/>
-			<var name="zxu"/>
-			<var name="zz"/>
-			<var name="zb"/>
-			<var name="zb3"/>
-			<var name="dss"/>
-			<var name="u_init"/>
-			<var name="v_init"/>
-			<var name="t_init"/>
-			<var name="qv_init"/>
-			<var name="deriv_two"/>
-			<var name="defc_a"/>
-			<var name="defc_b"/>
-			<var name="coeffs_reconstruct"/>
-			<var_array name="scalars"/>
-			<var name="initial_time"/>
-			<var name="xtime"/>
-			<var name="u"/>
-			<var name="w"/>
-			<var name="rho"/>
-			<var name="theta"/>
-			<var name="relhum"/>
-			<var name="rho_base"/>
-			<var name="theta_base"/>
-			<var name="surface_pressure"/>
-			<var name="isltyp"/>
-			<var name="ivgtyp"/>
-			<var name="mminlu"/>
-			<var name="isice_lu"/>
-			<var name="iswater_lu"/>
-			<var name="landmask"/>
-			<var name="shdmin"/>
-			<var name="shdmax"/>
-			<var name="snoalb"/>
-			<var name="albedo12m"/>
-			<var name="greenfrac"/>
-			<var name="sfc_albbck"/>
-			<var name="skintemp"/>
-			<var name="snow"/>
-			<var name="snowc"/>
-			<var name="snowh"/>
-			<var name="sst"/>
-			<var name="tmn"/>
-			<var name="vegfra"/>
-			<var name="seaice"/>
-			<var name="xice"/>
-			<var name="xland"/>
-			<var name="u10"/>
-			<var name="v10"/>
-			<var name="q2"/>
-			<var name="t2m"/>
-			<var name="precipw"/>
-			<var name="dzs"/>
-			<var name="sh2o"/>
-			<var name="smois"/>
-			<var name="tslb"/>
-			<var name="var2d"/>
-			<var name="con"/>
-			<var name="oa1"/>
-			<var name="oa2"/>
-			<var name="oa3"/>
-			<var name="oa4"/>
-			<var name="ol1"/>
-			<var name="ol2"/>
-			<var name="ol3"/>
-			<var name="ol4"/>
-			<var name="h_oml_initial"/>
+                        <var_array name="scalars"/>
+                        <var name="initial_time"/>
+                        <var name="xtime"/>
+                        <var name="cldfrac"/>
+                        <var name="re_cloud" packages="mp_thompson_in;mp_wsm6_in"/>
+                        <var name="re_ice" packages="mp_thompson_in;mp_wsm6_in"/>
+                        <var name="re_snow" packages="mp_thompson_in;mp_wsm6_in"/>
+                        <var name="u"/>
+                        <var name="w"/>
+                        <var name="pressure_p"/>
+                        <var name="pressure_base"/>
+                        <var name="rho"/>
+                        <var name="rho_base"/>
+                        <var name="theta"/>
+                        <var name="theta_base"/>
+                        <var name="relhum"/>
+                        <var name="uReconstructZonal"/>
+                        <var name="uReconstructMeridional"/>
+                        <var name="surface_pressure"/>
```

Comment:
Looks like there is one column misalignment, which leads to spurious diffs. Need to re-align those lines.